worlds/peaks_of_yore/docs/en_Peaks of Yore.md

```diff
@@ -0,0 +1,35 @@
+# Peaks of Yore
+
+## Where is the options page?
+The player options page for this game is located <a href="../player-options">here</a>. It contains all the options
```

Comment:
```suggestion
The player options page for this game is located [here](../player-options). It contains all the options
```
Mainly because most other games use this formatting